internal/server/server.go

```diff
@@ -100,11 +103,17 @@ func (s *server) Stop() {
 
 // ValidateCredential ensures that the given credential is a valid JWT issued by the OIDC issuer
 // the runtime was configured with.
-func (s *server) ValidateCredential(_ context.Context, req *authentication.ValidateCredentialRequest) (*authentication.ValidateCredentialResponse, error) {
+func (s *server) ValidateCredential(ctx context.Context, req *authentication.ValidateCredentialRequest) (*authentication.ValidateCredentialResponse, error) {
+	span := trace.SpanFromContext(ctx)
+
 	s.logger.Info("received CheckAccess request")
 
 	sub, claims, err := s.validator.ValidateToken(req.Credential)
 	if err != nil {
+		span.RecordError(err)
```

Comment:
One more thing:
```suggestion
		span.RecordError(err)
		span.SetStatus(tcodes.Error, err.Error())
```

Reply:
Actually, for consistency with checking access, we can leave this as-is for now.